UPGRADE-12.1.md

```diff
@@ -0,0 +1,47 @@
+# UPGRADING FROM 12.0 to 12.1
```

Comment:
I thought we want to keep upgrade from previous major and all current minors.

Reply:
added back